target/linux/qualcommax/ipq60xx/base-files/etc/board.d/01_leds

```diff
@@ -6,6 +6,11 @@ board_config_update
 board=$(board_name)
 
 case "$board" in
+8devices,mango-dvk|\
+8devices,mango-dvk-sfp)
+	ucidef_set_led_wlan "wlan5g" "WIFI 5GHz" "green:wlan5g" "phy0radio"
```

Comment:
There should be no reason for userspace trigger setting, you can set these via DT